MariaGKimani/Notes-Application
# Setup-django-project

### Description
This PR sets up the initial Django project structure, applies necessary migrations, and creates a superuser.

### Changes Made

- [x] Created a new Django project
- [x] Applied all migrations
- [x] Created a superuser for the admin panel

### Steps to Test
1. Pull the latest changes.
2. Run `python manage.py migrate` to ensure all migrations are applied.
3. Run `python manage.py createsuperuser` to create an admin user.
4. Start the server with `python manage.py runserver` and access the admin panel at `/admin`.
